applications/PoromechanicsApplication/python_scripts/poromechanics_MPI_U_Pw_solver.py

```diff
@@ -97,54 +35,76 @@ def AddVariables(self):
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.PARTITION_INDEX)
 
     def ImportModelPart(self):
-
         # Construct the Trilinos import model part utility
         import trilinos_import_model_part_utility
-        TrilinosModelPartImporter = trilinos_import_model_part_utility.TrilinosImportModelPartUtility(self.main_model_part, self.settings)
+        self.TrilinosModelPartImporter = trilinos_import_model_part_utility.TrilinosImportModelPartUtility(self.main_model_part, self.settings)
 
         # Execute the Metis partitioning and reading
-        TrilinosModelPartImporter.ExecutePartitioningAndReading()
+        self.TrilinosModelPartImporter.ExecutePartitioningAndReading()
```

Comment:
this fct is deprecated, it is now called "ImportModelPart" => #2285 
bcs it can now also do restart